### PR TITLE
Create no Story mdx document

### LINKED ISSUE #236: Replace Welcome story with a no-story MDX doc
**Description**
Currently the Welcome page of the Storybook is a story. It should be replaced with a [no-story MDX doc](https://github.com/storybookjs/storybook/blob/next/addons/docs/docs/mdx.md#documentation-only-mdx).

**Problems solved**
This opens the door for additional helpful pages such as 'Installation', 'Getting started' or 'How to use' etc. It'll also be an asset in the resolution of #200 by #214.

**Additional context**
The Story was a quick fix hack. This presents a chance to actually solve what was once hastily implemented as a temporary solution.

### PR DESCRIPTION
**Pull request by jahnvigupta**
## Fixes

Fixes #236 by @dhruvkb 

## Description

Replace Welcome Story Page with no story mdx Document.

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by jahnvigupta** [comment]
@dhruvkb just have a look on welcome.stories.mdx and ignore changes in package.json

**Comment by dhruvkb** [comment]
@jahnvigupta all your PRs are coming from a master that's much older than the `origin`. Please update the `master` branch on your fork.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Instead of creating a new story file, you should update the existing file `.storybook/welcome.stories.mdx`.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
... and needless to say, remove the changes to `package.json` from the PR.

File: .storybook/welcome.stories.mdx [hunk 1]
```diff
@@ -1,45 +1,26 @@
-import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import { Meta,Story,Preview } from '@storybook/addon-docs/blocks';
```
Comment (dhruvkb) [inline .storybook/welcome.stories.mdx]: Why did you remove the spaces?

File: .storybook/welcome.stories.mdx [hunk 2]
```diff
@@ -1,45 +1,26 @@
...
 
-<Meta title="Vocabulary/Welcome" />
+<Meta title="VOCABULARY/Intro"/>
```
Comment (dhruvkb) [inline .storybook/welcome.stories.mdx]: Why 'VOCABULARY' in all caps?

File: .storybook/welcome.stories.mdx [hunk 3]
```diff
@@ -1,45 +1,26 @@
...
       </p>
-      <a href="https://github.com/creativecommons/vocabulary" target="_blank">
+
+<a href="https://github.com/creativecommons/vocabulary" target="_blank">
         <button class="button is-primary">GitHub repository</button>
       </a>
-    </div>
-    <a class="netlify" href="https://www.netlify.com" target="_blank">
+</center>
+
```
Comment (dhruvkb) [inline .storybook/welcome.stories.mdx:23]: Point X.

File: .storybook/welcome.stories.mdx [hunk 4]
```diff
@@ -1,45 +1,26 @@
...
+<a class="netlify" href="https://www.netlify.com" target="_blank">
       <img src="https://www.netlify.com/img/global/badges/netlify-light.svg"/>
     </a>
-  `}</Story>
-</Preview>
 
 This is the Storybook for Vocabulary. Choose a component from the left panel to know more about it and explore the
 various customisations it has to offer.
```
Comment (dhruvkb) [inline .storybook/welcome.stories.mdx]: Move these lines to point X.

File: .storybook/welcome.stories.mdx [hunk 5]
```diff
@@ -1,45 +1,26 @@
-import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import { Meta,Story,Preview } from '@storybook/addon-docs/blocks';
 
-<Meta title="Vocabulary/Welcome" />
+<Meta title="VOCABULARY/Intro"/>
 
 Welcome to Vocabulary!
```
Comment (dhruvkb) [inline .storybook/welcome.stories.mdx]: Move this line to point X.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Some rearrangement is needed.

**Review by dhruvkb — APPROVED** [review]
Good to go.

**Comment by dhruvkb** [comment]
Changed 'Related to' to 'Fixes'
